README.md

```diff
@@ -13,14 +13,32 @@ This template includes setup for:
 
 ## How to Run
 
-The Next.js application is dockerized. Take a look at `./app/Dockerfile` to see how it works.
+### Without Docker
+
+You can run the Next.js app without docker as follows:
+
+1. `yarn install`
+2. `yarn dev`
+3. Navigate to `localhost:3000` in your browser to view the application
 
-A very simple `docker-compose.yml` has been included to support local development and deployment. Take a look at `./docker-compose.yml` for more information.
+You can run storybook without docker by running:
 
-How to run:
+1. `yarn storybook`
+2. Navigate to `localhost:6006` in your browser to view storybook
+
+### With Docker
+
+The Next.js application is dockerized. Take a look at `./app/Dockerfile` to see how it works.
+
+A `docker-compose.yml` has been included to support local development and deployment. Take a look at `./docker-compose.yml` for more information.
 
 1. In your terminal, `cd` to this repo.
 2. Make sure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed & running.
 3. Run `docker-compose up -d --build` to build the image and start the container.
-4. Navigate to `localhost:3000` in your browser to view the application.
-5. Run `docker-compose down` when you are done to delete the container.
+4. Navigate to `localhost:3000` in your browser to view the application. Note that it takes a few minutes for the initial sass compiling to complete and load.
+5. Run `docker-compose exec nextjs yarn storybook` to build and run storybook. Note that the initial sass compiling for storybook also takes a few minutes minutes to complete and load
```

Comment:
I still get the `Error: Loading PostCSS "postcss-preset-env" plugin failed: Cannot find module 'postcss-preset-env'` error when I run this 🙊

Reply:
Same comment as above. I'll think on this